2repository/6.2/enterprise-6.2.2/overload.gradle

```diff
@@ -0,0 +1,20 @@
+ext {
+    alfresco = [
+            version: [
+                    major: 6,
+                    minor: 2,
+                    rev: 2
+            ],
+            flavor: 'enterprise'
+    ]
+    solrimage = 'docker.io/xenit/alfresco-solr6:1.4.0'
```

Comment:
There is also an alfresco-solr:2.0.1 image. Do we need to use that one here instead of 1.4.0?

Reply:
alfresco-solr or alfresco-solr6?

Reply:
solr6

Reply:
recent build failed because the test pulls the image from docker hub, and 2x is not (yet?) published there. Considering the rate limiting, I can also just change the source to our harbor?